pkg/ovs/ovsconfig/interfaces.go

```diff
@@ -36,8 +36,8 @@ type OVSBridgeClient interface {
 	SetDatapathID(datapathID string) Error
 	GetInterfaceOptions(name string) (map[string]string, Error)
 	SetInterfaceOptions(name string, options map[string]interface{}) Error
-	CreatePort(name, ifDev string, externalIDs map[string]interface{}) (string, Error)
-	CreateInternalPort(name string, ofPortRequest int32, externalIDs map[string]interface{}) (string, Error)
+	CreatePort(name, ifDev string, antreaType string, externalIDs map[string]interface{}) (string, Error)
+	CreateInternalPort(name string, ofPortRequest int32, antreaType string, externalIDs map[string]interface{}) (string, Error)
```

Comment:
Before this patch, the interface is generic, all antrea specific logic are done by client side like converting K8s metadata to externalIDs and parsing from it. Can we follow it to avoid distributing the same logic on both client and API side?

Reply:
updated